src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,25 @@
 package mate.jdbc;
 
+import java.util.ArrayList;
+import java.util.List;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Driver;
+import mate.jdbc.service.DriverService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        List<Driver> driverList = new ArrayList<>();
+        driverList.add(new Driver(4L, "Den", "19875"));
```

Comment:
id will be generated in DB automatically. its redundant parameter in the constructor. create a constructor with two parametrs